website/docs/installation/storage-s3.md

```diff
@@ -79,10 +79,18 @@ AUTHENTIK_STORAGE__MEDIA__S3__ENDPOINT=https://s3.provider
 AUTHENTIK_STORAGE__MEDIA__S3__CUSTOM_DOMAIN=s3.provider/authentik-media
 ```
 
+If you are using Backblaze B2 as your S3 provider and the bucket is set to `Public`, add the following:
```

Comment:
If the bucket is set to `Public` it only allows creation of files with the `public-read` ACL, same with `Private` which requires the `private` ACL. When I set my bucket to `Private`, it would allow authentik to upload the files but I couldn't view them. When it was set to `Public`, it wouldn't allow authentik to upload the files without this change.
![image](https://github.com/goauthentik/authentik/assets/68230846/041313d4-e633-42db-9af1-fe33c29b5797)

Reply:
@rissson I just realised my mistake in the PR description and will correct it. "Set as `private`" is meant to be "Set as `public`".

Reply:
Ah that's a different issue. The bucket needs to be made private then, otherwise anyone would be able to access those files.

Reply:
For media files, it isn't a real issue as they'll likely just be icons. But we might use that storage later on for other files that shouldn't be made public.

Reply:
Ah. Considering that, should I close the PR and use something other than Backblaze? Since the only other way to use Backblaze would be to use something like Cloudflare Workers which I believe would incur an additional cost: https://www.backblaze.com/docs/cloud-storage-deliver-private-backblaze-b2-content-through-cloudflare-cdn.

Reply:
It should still work, we serve the bucket files with pre-signed URLs (https://www.backblaze.com/docs/cloud-storage-s3-compatible-api#pre-signed-urls)